hack/jenkins/e2e.sh

```diff
@@ -66,15 +66,50 @@ if [[ "${KUBERNETES_PROVIDER}" == "aws" ]]; then
 fi
 
 # Specialized tests which should be skipped by default for projects.
-GCE_DEFAULT_SKIP_TEST_REGEX="Skipped|Density|Reboot|Restart|Example"
+GCE_DEFAULT_SKIP_TEST_REGEX="\
+Skipped\
+|Density\
+|Reboot\
+|Restart\
+|Example\
+"
 # The following tests are known to be flaky, and are thus run only in their own
 # -flaky- build variants.
-GCE_FLAKY_TEST_REGEX="Addon|Elasticsearch|Nodes.*network\spartition|Proxy.*logs|Shell.*services|MaxPods.*"
+GCE_FLAKY_TEST_REGEX="\
+Addon\
```

Comment:
Can you remove Addon - I moved it out of flaky this morning (it didn't fail since quite long).